src/main/java/stream/flarebot/flarebot/util/general/GeneralUtils.java

```diff
@@ -137,28 +133,242 @@ public static String getProgressBar(Track track) {
+        List<T> added = new ArrayList<>();
+        for (T oldItem : oldList) {
+            if (!newList.contains(oldItem)) {
+                removed.add(oldItem);
+            }
+        }
+        for (T newItem : newList) {
+            if (!oldList.contains(newItem)) {
+                added.add(newItem);
+            }
+        }
+        changes.put(true, added);
+        changes.put(false, removed);
+        return changes;
     }
 
-    public static String formatCommandPrefix(GuildWrapper guild, String usage) {
-        String prefix = String.valueOf(getPrefix(guild));
-        if (usage.contains("{%}"))
-            return usage.replaceAll("\\{%}", prefix);
-        return usage;
+    /**
+     * Gets the suffix the a day in a month
+     *
+     * @param n The day in the month to get a suffix.
+     * @return The suffix for they day.
+     */
+    public static String getDayOfMonthSuffix(final int n) {
+        if (n < 1 || n > 31) throw new IllegalArgumentException("illegal day of month: " + n);
+        if (n >= 11 && n <= 13) {
+            return "th";
+        }
+        switch (n % 10) {
+            case 1:
+                return "st";
+            case 2:
+                return "nd";
+            case 3:
+                return "rd";
+            default:
+                return "th";
+        }
     }
 
+    /**
+     * Message fields:
+     * - Message ID
+     * - Author ID
+     * - Channel ID
+     * - Guild ID
+     * - Raw Content
+     * - Timestamp (Epoch seconds)
+     *
+     * @param message The message to serialise
+     * @return The serialised message
+     */
+    public static String getRedisMessageJson(Message message) {
+        return FlareBot.GSON.toJson(new RedisMessage(
+                message.getId(),
+                message.getAuthor().getId(),
+                message.getChannel().getId(),
+                message.getGuild().getId(),
+                message.getContentRaw(),
+                message.getCreationTime().toInstant().toEpochMilli()
+        ));
+    }
+
+    /**
+     * Gets the String representing the verification level.
+     *
+     * @param level The verification level to get the string version.
+     * @return A string representing the verification level.
+     */
+    public static String getVerificationString(Guild.VerificationLevel level) {
+        switch (level) {
+            case HIGH:
+                return "(\u256F\u00B0\u25A1\u00B0\uFF09\u256F\uFE35 \u253B\u2501\u253B"; //(╯°□°）╯︵ ┻━┻
+            case VERY_HIGH:
+                return "\u253B\u2501\u253B\u5F61 \u30FD(\u0CA0\u76CA\u0CA0)\u30CE\u5F61\u253B\u2501\u253B"; //┻━┻彡 ヽ(ಠ益ಠ)ノ彡┻━┻
+            default:
+                return level.toString().charAt(0) + level.toString().substring(1).toLowerCase();
+        }
+    }
+
+    /**
+     * Parses time from string.
+     *
+     * @param time The time string to parse.
+     * @return The long representing the time entered.
```

Comment:
Is this a Java Epoch long? Long as in years since it happened? Days? Is it how long it took to parse? More info